Makefile

```diff
@@ -307,12 +307,39 @@ $(BUILD_DIRECTORY)/$(APPLICATION_NAME_ELECTRON)_$(APPLICATION_VERSION_DEBIAN)_$(
 		--extraMetadata.packageType=deb \
 		$(DISABLE_UPDATES_ELECTRON_BUILDER_OPTIONS)
 
-$(BUILD_DIRECTORY)/$(APPLICATION_NAME_LOWERCASE)-$(APPLICATION_VERSION)-$(TARGET_ARCH_APPIMAGE).AppImage: \
-	| $(BUILD_DIRECTORY)
-	$(NPX) build --linux AppImage $(ELECTRON_BUILDER_OPTIONS) \
+ifeq ($(TARGET_ARCH),x64)
+ELECTRON_BUILDER_LINUX_UNPACKED_DIRECTORY = linux-unpacked
+else
+ELECTRON_BUILDER_LINUX_UNPACKED_DIRECTORY = linux-$(TARGET_ARCH_ELECTRON_BUILDER)-unpacked
+endif
+
+$(BUILD_DIRECTORY)/$(ELECTRON_BUILDER_LINUX_UNPACKED_DIRECTORY):
```

Comment:
This ought to have an order-only dependency on `$(BUILD_DIRECTORY)` (i.e. add `	| $(BUILD_DIRECTORY)` to the end of the line)